pkg/odo/cli/component/create.go

```diff
@@ -315,6 +329,7 @@ func ensureAndLogProperResourceUsage(resource, resourceMin, resourceMax, resourc
 func NewCmdCreate() *cobra.Command {
 	componentCreateCmd.Flags().StringVarP(&componentBinary, "binary", "b", "", "Use a binary as the source file for the component")
 	componentCreateCmd.Flags().StringVarP(&componentGit, "git", "g", "", "Use a git repository as the source file for the component")
+	componentCreateCmd.Flags().StringVarP(&componentGitRef, "ref", "r", "", "Use a specific ref i.e. commit, branch or tag of the git repository")
```

Comment:
use `ex` instead of `i.e.`

Reply:
`e.g.` is the more appropriate term, I believe ;)

Reply:
ack